PyISY/Programs/__init__.py

```diff
@@ -192,7 +192,9 @@ def parse(self, xml):
                             plastrun = _empty_time
                         else:
                             plastrun = datetime.strptime(
-                                plastrun.toxml(), '%Y/%m/%d %I:%M:%S %p')
+                                plastrun.toxml(), '%Y/%m/%d %I:%M:%S %p'
+                                if plastrun.toxml().count(' ') == 2

```

Comment:
Please add a comment about what this check is doing so it's easier to maintain in the future :)

Reply:
Please see if the last push is a little more clear on what the check is doing--basically all it does is sees if there are 2 spaces in the time string which would mean it has an AM/PM on the end, if there's only 1 space, then it's military/24-hr time.